composer.lock

```diff
@@ -0,0 +1,5177 @@
+{
```

Comment:
This `composer.lock` needs to be re-generated to a version compatible with all tested php versions

Reply:
See https://github.com/laminas/laminas-skeleton-installer/pull/26 for an attempt to add composer.lock

Reply:
@Ocramius updated the lock to `PHP 7.3`. 

`phpcs` is failure should be addressed in a separate PR.

Reply:
I'd suggest merging #26 before this one, because it restore CI entirely, including phpcs.